src/containers/PageHeader/__tests__/PageHeader.test.js

```diff
@@ -52,6 +52,12 @@ describe('containers/PageHeader', () => {
     ));
 
     expect(container.firstChild.querySelector('h1').textContent).toEqual('Foo bar !!1! (99)');
+
+    rerender(withAppContext(
+      <PageHeaderContainerComponent filter={{ name: 'Foo bar !!1!', refresh: true }} incidentsCount={99} />
+    ));
+
+    expect(container.firstChild.querySelector('svg')).toBeTruthy();
```

Comment:
is checken of alleen 1 svg in de header zit niet te weinig specifiek voor deze test?

Reply:
What else what you suggest? The only change to this component is the rendering of the svg icon.